uniprot-config/src/main/java/org/uniprot/store/config/searchfield/model/SearchFieldItem.java

```diff
@@ -29,6 +29,7 @@ public class SearchFieldItem implements Field, Serializable {
     private String example;
     private String validRegex;
     private List<String> tags;
+    private List<String> aliases = List.of();
```

Comment:
Can you please let me know why do we need to initialise this one? (I am asking because it is inconsistent with others)

Reply:
to avoid NPE while streaming, ex: AbstractSeachFieldConfid: 73

https://github.com/ebi-uniprot/uniprot-store/blob/b331e53e1b62b0da447d92bebff04c9d9544338b/uniprot-config/src/main/java/org/uniprot/store/config/searchfield/common/AbstractSearchFieldConfig.java#L73